### PR TITLE
update performance evaluation form layout

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Performance Evaluation Form Changes#159](https://github.com/nerubia/kh360-react/issues/159)

### CODE REVIEW

File: src/utils/formatDate.ts [hunk 1]
```diff
@@ -1,3 +1,40 @@
 export const formatDate = (date?: string) => {
   return date?.split("T")[0]
 }
+
+export const formatDateRange = (start_date?: string, end_date?: string) => {
```
Comment (jatnerubia) [inline src/utils/formatDate.ts:5]: I  also have an update for this. Can you check to see if we can reuse it. [here](https://github.com/nerubia/kh360-react/pull/165/files#diff-3290e87c24e27923abe4a89682818bc0169238f9c4784ab4545dd448c0951415)

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/features/evaluations/[id]/EvaluationsCriteria.tsx [hunk 2]
```diff
@@ -166,11 +166,28 @@ export const EvaluationsCriteria = () => {
       {loading === Loading.Fulfilled &&
         evaluation_template_contents !== null &&
         user_evaluations.length > 0 && (
-          <div className='flex flex-col overflow-y-scroll pr-5 mr-4 w-3/4'>
-            <div className='text-lg font-bold text-primary-500 mb-2'>
-              <h1>Project Assignment Duration</h1>({formatDate(evaluation?.eval_start_date)} to{" "}
-              {formatDate(evaluation?.eval_end_date)})
+          <div className='flex flex-col overflow-y-scroll pr-5 mx-4 md:w-3/4'>
+            <div className='text-xl font-bold text-primary-500 mb-1'>
+              <h1>
```
Comment (jatnerubia) [inline src/features/evaluations/[id]/EvaluationsCriteria.tsx:171]: we should have one `h1` per page

File: src/features/evaluations/[id]/EvaluationsCriteria.tsx [hunk 3]
```diff
@@ -166,11 +166,28 @@ export const EvaluationsCriteria = () => {
...
+                {evaluation.template?.project_role?.name !== undefined
+                  ? evaluation.template?.project_role?.name
+                  : evaluation.template?.display_name ?? ""}
+              </p>
+            ) : (
+              <p className='text-base font-bold mb-1'>
+                {evaluation?.project?.name} [{evaluation?.project_role?.short_name}]
+              </p>
+            )}
+            <h1 className='mb-4 text-sm'>
```
Comment (jatnerubia) [inline src/features/evaluations/[id]/EvaluationsCriteria.tsx:187]: we should have one `h1` per page

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Review by jatnerubia — APPROVED** [review]
Nice

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**